eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/SendLinkHandler.java

```diff
@@ -73,7 +72,7 @@ public void onDelivery(Event event) {
                 TRACE_LOGGER.trace(
                         "onDelivery linkName[" + sender.getName()
                                 + "], unsettled[" + sender.getUnsettled() + "], credit[" + sender.getRemoteCredit() + "], deliveryState[" + delivery.getRemoteState()
-                                + "], delivery.isBuffered[" + delivery.isBuffered() + "], delivery.id[" + new String(delivery.getTag(), UTF_8) + "]");
+                                + "], delivery.isBuffered[" + delivery.isBuffered() + "], delivery.id[" + new String(delivery.getTag(), StandardCharsets.UTF_8) + "]");
```

Comment:
@conniey @g2vinay @mssfang I propose we change our checkstyle rules here - I like static imports as it was before, and would not like to have to enforce that we do not use them. Thoughts?

Reply:
I dont mind relaxing this rule. makes it easier to read, too.

Reply:
agree. It is more clear to read.

Reply:
@mssfang please revert the changes where you got rid of the static import such as here. Once you've done that, please let me know what CheckStyle tests fail and we can work to modify that.

Reply:
I have made my changes for static import at commit "refactor(SpotBugs): import static"
checkstyle changed at from 
"fix<checkstyle>: access can be package-private"   to 
"fix<checkstyle>: modifier final is redundant for final class"

Reply:
Do you not now get a report from CheckStyle about these 'issues' returning? It doesn't appear that you changed the CheckStyle rules. If there are no issues being reported now (after your revert), why was this changed in the first place?

Reply:
The reason I changed from import static to import at beginning is because I added import static at first place and then I saw other existing file using import instead of import static. I want to keep coding style consistency. There is no error introduced at first space.  Sorry for causing the confusion.